### DISCUSSION TITLE
Welcome to GAN-for-tabular-data Discussions!

### DISCUSSION THREAD

**Discussion by Diyago** [Diyago/Tabular-data-generation#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jfbldevs** [comment]
Excellent project, could you make an example starting from a .csv file?

**Comment by Diyago** [comment]
Sure! Example shown on the readme page
